### PR TITLE
How to specify an engine's URI location

### PR DESCRIPTION
**Pull request by meori**
I'll try and document obstacles I tackle while I implement engines for our app.

### CODE REVIEW

File: markdown/guide/deploying-engines.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Deploying An Engine
+
+In many cases, the engine code will be served from a static server, typically a CDN.
```
Comment (trentmwillis) [inline markdown/guide/deploying-engines.md]: We keep paragraphs on a single line, so can you combine these down?

File: markdown/guide/deploying-engines.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+This server will probably not share the same root location as your application.
+In order to specify the engine's assets location, you can implement a `generateURI` function at the host application's build options file:
```
Comment (trentmwillis) [inline markdown/guide/deploying-engines.md]: The phrase "engine's assets location" reads a little odd to me, can we rephrase just as "engine asset location"?

Also, let's change "at the host application's build options file" to "in the host application's `ember-cli-build.js`"

File: markdown/guide/deploying-engines.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      if (EmberApp.env() === 'production') {
+        return '//production.cdn.com/' + filePath;
+      } else {
+        return 'local/static/' + filePath;
+      }
+    }
+  }
+});
+```
+The `generateURI` function receives the `filePath` for each asset and must return a string of the asset's location.
```
Comment (trentmwillis) [inline markdown/guide/deploying-engines.md]: Need a newline before this paragraph. Also, let's add a link to the `ember-asset-loader` documentation here. Maybe: `For more info, see the [ember-asset-loader documentation](https://github.com/ember-engines/ember-asset-loader#generating-custom-uris)`

File: markdown/guide/deploying-engines.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (trentmwillis) [inline markdown/guide/deploying-engines.md:19]: Should delete this line.

**Review by trentmwillis — CHANGES_REQUESTED** [review]
Looking good, thanks for the PR! Some minor tweaks I'd like to see before merging.

**Comment by meori** [comment]
Fixed the text. Hope it works for you :)

**Review by trentmwillis — APPROVED** [review]

**Pull request merged**